doc/source/rest.j2

```diff
@@ -532,112 +663,71 @@ into by using the `metric_id` query parameter several times.
 
 {{ scenarios['search-value-in-metric']['doc'] }}
 
-And it is possible to search for values in |metrics| by using one or multiple
+And it is possible to search for values in |metrics| by using one or more
 |granularities|:
 
 {{ scenarios['search-value-in-metrics-by-granularity']['doc'] }}
 
-You can specify a time range to look for by specifying the `start` and/or
-`stop` query parameter, and the |aggregation method| to use by specifying the
-`aggregation` query parameter.
-
-The supported operators are: equal to (`=`, `==` or `eq`), lesser than (`<` or
-`lt`), greater than (`>` or `gt`), less than or equal to (`<=`, `le` or `≤`)
-greater than or equal to (`>=`, `ge` or `≥`) not equal to (`!=`, `ne` or `≠`),
-addition (`+` or `add`), substraction (`-` or `sub`), multiplication (`*`,
-`mul` or `×`), division (`/`, `div` or `÷`). These operations take either one
-argument, and in this case the second argument passed is the value, or it.
-
-The operators or (`or` or `∨`), and (`and` or `∧`) and `not` are also
-supported, and take a list of arguments as parameters.
 
 .. _aggregates:
 
-Aggregates: on the fly, measurements modification and aggregation
-===================================================================
+Aggregates
```

Comment:
`on the fly` should be retained. Otherwise, it'll be confused with the stored `Aggregates` above.